docs/content/5.integrations/_integrations.json

```diff
@@ -26,6 +26,12 @@
       "title": "Salesforce Commerce Cloud",
       "to": "/integrations/sfcc",
       "icon": "logos:salesforce"
+    },
+    {
+      "title": "Plentymarkets",
+      "to": "https://pwa-docs.plentymarkets.com/",
+      "icon": "IconPlentyMarkets",
+      "partnerMaintained": true
```

Comment:
at the moment, this won't do anything - but it would be a nice-to-have for future enhancements

Reply:
I'm not sure what you mean. What won't do anything?